src/control/lib/netdetect/netdetect.go

```diff
@@ -141,21 +144,22 @@ const (
 	allHFIUsed            = -1
 	badAddress            = C.getHFIUnitError
 	// ARP protocol hardware identifiers: https://elixir.free-electrons.com/linux/v4.0/source/include/uapi/linux/if_arp.h#L29
-	Netrom     = 0
-	Ether      = 1
-	Eether     = 2
-	Ax25       = 3
-	Pronet     = 4
-	Chaos      = 5
-	IEEE802    = 6
-	Arcnet     = 7
-	Appletlk   = 8
-	Dlci       = 15
-	Atm        = 19
-	Metricom   = 23
-	IEEE1394   = 24
-	Eui64      = 27
-	Infiniband = 32
+	Netrom          = 0
+	Ether           = 1
+	Eether          = 2
+	Ax25            = 3
+	Pronet          = 4
+	Chaos           = 5
+	IEEE802         = 6
+	Arcnet          = 7
+	Appletlk        = 8
+	Dlci            = 15
+	Atm             = 19
+	Metricom        = 23
+	IEEE1394        = 24
+	Eui64           = 27
+	Infiniband      = 32
+	topologyKey key = 0
```

Comment:
I'd be inclined to either move this declaration above the network type enum, or break out the network type enum list into its own `const` block just for clarity.

Reply:
Done.